src/System/Wlog/Severity.hs

```diff
@@ -71,3 +78,6 @@ errorPlus = Set.singleton Error
 -- | Excludes 'Error' from the 'Set' of 'Severity's.
 excludeError :: Set Severity -> Set Severity
 excludeError = Set.delete Error
+
+unionSeverities :: Severities -> Severities -> Severities
+unionSeverities = Set.union
```

Comment:
I think we don't need `unionSeverities` function because `Set` is `Semigroup` and has `<>` operator [which is implemented](http://hackage.haskell.org/package/containers-0.5.10.2/docs/src/Data.Set.Internal.html#line-286) just like `union`.